modules/microshift-install-rpms-gitops.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-installing-rpms-for-gitops_{context}"]
+== Installing the GitOps ArgoCD manifests from an RPM package
+
+You can use a lightweight version of the OpenShift GitOps with {microshift-short} to help manage your applications. Install the necessary ArgoCD manifests using an RPM package. 
+
+.Procedure
+
+. Enable the GitOps repository with the subscription manager by running the following command:
++
+----
+$ sudo subscription-manager repos --enable=gitops-1.12-for-rhel-9-$(uname -m)-rpms
+----
+
+. Install the GitOps packages by running the following command: 
++
+[source,terminal]
+----
+$ sudo dnf install -y microshift-gitops microshift-gitops-release-info
```

Comment:
No need to install the release-info package. That is only required for offline/airgapped installation to determine which container images are required by gitops. Just "sudo dnf install -y microshift-gitops" is enough

Reply:
Ah gotcha! Thank you!